user-data-store-client-model/src/main/java/com/wultra/security/userdatastore/client/UserDataStoreClient.java

```diff
@@ -87,6 +84,15 @@ public interface UserDataStoreClient {
      */
     PhotoCreateResponse createPhoto(PhotoCreateRequest request) throws UserDataStoreClientException;
 
+    /**
+     * Update a photo.
+     *
+     * @param photoId Photo identifier.
+     * @param request Photo update request.
+     * @throws UserDataStoreClientException Thrown in case REST API call fails.
+     */
+    void updatePhoto(String photoId, PhotoUpdateRequest request) throws UserDataStoreClientException;
```

Comment:
🤔 Just because of a compatibility reason, what about returning the response class, even though it is an empty envelope so far?

Reply:
This would be inconsistent, the client does not use `Response` or `ObjectRequest`/`ObjectResponse` classes.

See e.g. `PhotoUpdateRequest request`, not `ObjectRequst<PhotoUpdateRequest> request`.